app/models/accounting/quickbooks/updater.rb

```diff
@@ -88,16 +87,7 @@ def extract_qb_data(txn)
 
         extractor = Accounting::Quickbooks::DataExtractor.new(txn)
         extractor.extract!
```

Comment:
Do the changes actually stick when you do this? I wouldn't expect the txn object to have actually been mutated with all the changes without saving it inside the extractor.

Reply:
I'll write a spec to cover that